export.js

```diff
@@ -37,25 +37,51 @@ if(platform == "linux"){
     options.executablePath = "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"
 }
 
-
 async function main() {
     try{
         if(platform == "linux"){
             xvfb.startSync()
         }
-        var url = process.argv[2],
-            exportname = process.argv[3], 
-            duration = process.argv[4],
-            convert = process.argv[5]
-
-        if(!url){ url = 'https://www.mynaparrot.com/' }
-        if(!exportname){ exportname = 'export.webm' }
-        if(!duration){ duration = 10 }
-        if(!convert){ convert = false }
         
+        var url = process.argv[2];
+        if(!url){
+            console.warn('URL undefined!');
+            process.exit(1);
+        }
+        // Verify if recording URL has the correct format
+        var urlRegex = new RegExp('^https?:\\/\\/.*\\/playback\\/presentation\\/2\\.0\\/playback\\.html\\?meetingId=[a-z0-9]{40}-[0-9]{13}');
+        if(!urlRegex.test(url)){
+            console.warn('Invalid recording URL!');
+            process.exit(1);
+        }
+
+        var exportname = process.argv[3];
+        // Use meeting ID as export name if it isn't defined or if its value is "meetingid"
+        if(!exportname || exportname == "meetingid"){
```

Comment:
I don't understand why you have used `meetingid` here. It could be meeting or better remove it. Let it be same as example.

Reply:
OK, I will change the default value to "MEETING_ID" and the reserved word from "meetingid" to "MEETING_ID" to unify criteria.